libs/langchain/langchain/document_loaders/pubmed.py

```diff
@@ -0,0 +1,29 @@
+from typing import Iterator, List, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+from langchain.utilities.pubmed import PubMedAPIWrapper
+
+
+class PubMedLoader(BaseLoader):
+    """Loads a query result from PubMed biomedical library into a list of Documents."""
```

Comment:
```suggestion
    """Loads a query result from PubMed biomedical library into a list of documents."""
```

Reply:
It is a class name, a specific class, not just a generic document.

Reply:
yeah makes sense -- i favor documenting concepts rather than types except for in Args / Returns sections. This is a personal preference, no need to implement follow :)